facebook-messenger/build.gradle

```diff
@@ -42,6 +42,7 @@ android {
         targetSdkVersion project.ext.targetSdk
         consumerProguardFiles 'proguard-rules.pro'
         vectorDrawables.useSupportLibrary = true
+        resourcePrefix "messenger_"
```

Comment:
I'm not sure why are you guys decided to choose another prefix to the messenger module is there any reason other than that you didn't want to make a breaking change?